Sources/GryphonLib/Driver.swift

```diff
@@ -39,6 +39,7 @@ public class Driver {
 	public static let supportedArgumentsWithParameters: List = [
 		"--indentation=",
 		"--target=",
+        "--UseModernBuildSystem",
```

Comment:
It might be better to put this in the `supportedArguments`, it doesn't look like this needs a parameter... if the argument is included we consider it to be `"YES"`, otherwise it's `"NO"`, right?

Reply:
Also, don't forget to add a description in the `usageString` later so people know how to use your new argument. It might be good to mention there that the compilation might take a while.